plugins/item_previews/web_client/stylesheets/item_previews.styl

```diff
@@ -0,0 +1,40 @@
+maxPreviewHeight = 400px
```

Comment:
By convention we like to start our stylus variables with the $ character to make it obvious that they are variables rather than raw values.